CHANGELOG.md

```diff
@@ -5,6 +5,19 @@ For 1.x changes, go [here](https://github.com/twilio/twilio-video.js/blob/suppor
 
 New Features
 ------------
+- By default, twilio-video.js connects to your nearest signaling server, as determined by
+  [latency based routing](https://www.twilio.com/docs/video/ip-address-whitelisting#signaling-communication).
+  You can now override this behavior by using a new ConnectOptions flag called `region`. This will make
+  sure that your signaling traffic will terminate at the specified region. (JSDK-2338)
+
+  ```js
+  const { connect } = require('twilio-video');
+  const room = await connect(token, {
+    region: 'de1'
+  });
+  ```
```

Comment:
@makarandp0 Nitpick: Can you add a new line here?

Reply:
@aymenn can you take a look at this changelog entry please, and suggest any updates.

Reply:
Suggestion: "terminate at the" -> "terminate in the"

Overall, this sounds good to me, specifically leaving out mention of "gll" media behavior which is not ready yet. We will need to update the white listing guide to mention the us1, de1, and in1 VSS deployments sometime soon.

Reply:
updated per your suggestion @ceaglest.